apis/r/NEWS.md

```diff
@@ -1,3 +1,7 @@
+# 1.9.4
+
+* This release contains an in-progress blockwise iterator, work on which will be completed in a subsequent release.
```

Comment:
You could mention the other PRs that landed that reverted `nanoarrow` use to vendored versions and refactor the use throughout R and adjusted the build steps accordingly.

Reply:
@eddelbuettel https://github.com/single-cell-data/TileDB-SOMA/pull/2399/commits/e4b0f2845f34b73033eb12595a7a53b08f78e424 thank you